corehq/form_processor/backends/sql/processor.py

```diff
@@ -210,56 +210,75 @@ def submission_error_form_instance(cls, domain, instance, message):
     def get_cases_from_forms(case_db, xforms):
         """Get all cases affected by the forms. Includes new cases, updated cases.
         """
-        touched_cases = {}
-        if len(xforms) > 1:
-            domain = xforms[0].domain
+        if len(xforms) == 1:
```

Comment:
what happens if len(xforms) is 0. this appears to change that behavior

Reply:
that would previously have errored when running `xforms[0]`